java/integTest/src/test/java/glide/TestUtilities.java

```diff
@@ -394,21 +394,35 @@ public static String createLongRunningLuaScript(int timeout, boolean readOnly) {
         return script.replace("$timeout", Integer.toString(timeout));
     }
 
-    public static void waitForNotBusy(BaseClient client) {
+    /**
+     * Lock test until server completes a script/function execution.
+     *
+     * @param function true if need to kill a function, false to kill a script.
+     */
+    @SneakyThrows
+    public static void waitForNotBusy(BaseClient client, boolean function) {
```

Comment:
pass a lambda function to be executed - not a boolean.  This would make it universal for any "kill-like" command.